mix.exs

```diff
@@ -0,0 +1,64 @@
+defmodule PLDS.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :plds,
+      version: "0.1.0",
+      elixir: "~> 1.12",
+      elixirc_paths: elixirc_paths(Mix.env()),
+      compilers: Mix.compilers(),
+      start_permanent: Mix.env() == :prod,
+      aliases: aliases(),
+      escript: escript(),
+      deps: deps()
+    ]
+  end
+
+  # Configuration for the OTP application.
+  #
+  # Type `mix help compile.app` for more information.
+  def application do
+    [
+      mod: {PLDS.Application, []},
+      extra_applications: [:logger, :runtime_tools, :os_mon, :inets]
+    ]
+  end
+
+  # Specifies which paths to compile per environment.
+  defp elixirc_paths(:test), do: ["lib", "test/support"]
+  defp elixirc_paths(_), do: ["lib"]
+
+  # Specifies your project dependencies.
+  #
+  # Type `mix help deps` for examples and options.
+  defp deps do
+    [
+      {:phoenix, "~> 1.6.0-rc.0", override: true},
+      {:phoenix_live_dashboard, path: "../phoenix_live_dashboard", override: true},
+      {:telemetry_metrics, "~> 0.6"},
+      {:ecto_psql_extras, "~> 0.7"},
+      {:broadway_dashboard, "~> 0.2.1"},
+      {:jason, "~> 1.2"},
+      {:plug_cowboy, "~> 2.5"}
+    ]
+  end
```

Comment:
I'm not sure if you're ready to do so yet, or not, but this is what we do to make the LiveView dependency configurable on the dashboard:

```suggestion
      {:phoenix_live_dashboard, phoenix_live_dashboard_opts()},
      {:telemetry_metrics, "~> 0.6"},
      {:ecto_psql_extras, "~> 0.7"},
      {:broadway_dashboard, "~> 0.2.1"},
      {:jason, "~> 1.2"},
      {:plug_cowboy, "~> 2.5"}
    ]
  end

  defp phoenix_live_dashboard_opts do
    if path == System.get_env("LIVE_DASHBOARD_PATH") do
      [path: path, override: true]
    else
      []
    end
  end
```

but it's just a thought! :)

Reply:
@mcrumm cool! Thanks! And yeah, I think this is temporary, but can be handy in the future. :smile: 
I will just rebase and add your suggestion after finishing adding some tests.